applications/asset_tracker_v2/src/modules/Kconfig.cloud_module

```diff
@@ -32,12 +32,28 @@ choice NRF_CLOUD_SEND_SOCKET
 	default NRF_CLOUD_SEND_TIMEOUT
 endchoice
 
-# MQTT Transport library
-config MQTT_CLEAN_SESSION
-	default y
+choice NRF_CLOUD_CLIENT_ID_SRC
+	default NRF_CLOUD_CLIENT_ID_SRC_RUNTIME
+endchoice
+
+if BOARD_QEMU_X86
+
+config NRF_CLOUD_FOTA
+	bool ""
+	default n
+
+endif # BOARD_QEMU_X86
 
 endif # NRF_CLOUD_MQTT
 
+if AZURE_FOTA_TLS
+
+config AZURE_FOTA_SEC_TAG
```

Comment:
Ditto.

Reply:
I moved this to `Kconfig.azure_fota_patch`. Setting this Kconfig in the `overlay-azure.conf` (where it previously was set) file doesn't work because the Kconfig is not visible for QEMU x86 builds when Azure FOTA is disabled. It has to be set in the cases Azure FOTA is enabled to prevent the build assert in https://github.com/simensrostad/fw-nrfconnect-nrf/blob/9c79728d9daa63264764dd844c252693334684a7/subsys/net/lib/azure_fota/azure_fota.c#L82.